spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/JGitEnvironmentRepository.java

```diff
@@ -16,12 +16,6 @@
 
 package org.springframework.cloud.config.server.environment;
 
-import java.io.File;
```

Comment:
I dont think these import reorgs are correct.  Try using the eclipse settings in Spring Cloud Netflix to make sure you follow the correct import order standards https://github.com/spring-cloud/spring-cloud-netflix/tree/master/eclipse